src/App.js

```diff
@@ -50,28 +50,29 @@ function App() {
   return (
     <div className="App">
       <Router>
-        <div>
-          <nav className="d-flex align-items-center justify-content-between">
-            <h1 className="px-5"><b>To Do List</b></h1>
-            <ul className="d-flex list-unstyled mx-5">
-              <li className="mx-5">
-                <Link to="/">Books</Link>
+        <div className="container-fluid px-5">
```

Comment:
- [ ] Kindly deploy the project online and include the live deploy link in the readme.